.github/workflows/ci-windows-installed.yml

```diff
@@ -0,0 +1,76 @@
+# This is a copy of ci-windows.yml for installed dependencies. It defers in the following ways:
+#
+#   * vcpkg is used to install dependencies that are packaged.
+#   * rav1e is installed from a binary upstream.
+#   * TODO: use proper installations of libgav1, libsharpyuv and SVT once released.
+
+name: CI Windows Installed
+on: [push, pull_request]
```

Comment:
Note for another PR: The number of CI workflows if growing. Maybe we could have some on `push-to-main` only to keep resource usage and presubmit duration low enough.

Reply:
The workflows with system installed packages are much faster to run. Rather than restricting workflows by event or branch you could consider conditioning local dependency install workflows on changes to the cmake files and ext scripts.

Reply:
Done

Reply:
I am proposing https://github.com/AOMediaCodec/libavif/pull/2115 to limit the slow jobs to such conditions. PTAL.